Library/Formula/translate-shell.rb

```diff
@@ -0,0 +1,31 @@
+class TranslateShell < Formula
+  desc "Command-line translator using Google Translate and more"
+  homepage "https://www.soimort.org/translate-shell"
+  url "https://github.com/soimort/translate-shell/archive/v0.9.3.1.tar.gz"
+  sha256 "7b5a2404ead919570cfa4d741c521ed7e124bbf27d6f5de1a6a598e7e713c2a6"
+  head "https://github.com/soimort/translate-shell.git", :branch => "develop"
+
+  option "with-espeak", "Use eSpeak for Text-to-Speech"
+  option "with-mpg123", "Use mpg123 for Text-to-Speech"
+  option "with-mplayer", "Use mplayer for Text-to-Speech"
+  option "with-mpv", "Use mpv for Text-to-Speech"
+
+  depends_on "fribidi"
+  depends_on "gawk"
+  depends_on "rlwrap"
+
+  depends_on "espeak" => :optional
+  depends_on "mpg123" => :optional
```

Comment:
As you can see from my comment, I wasn't sure about how to approach this. Maybe we can just strip them all, since they are just optional runtime deps that can be installed after the fact, and users concerned should find out by reading the project's README? Or maybe add a caveat? If you have an idea I'm all ears.